src/utils/validation/input-validation.ts

```diff
@@ -15,14 +15,15 @@ export function inputDateOfBirthValidation(event: Event, errorMessage: string):
       const inputDate = input.value;
       const birthDate = new Date(inputDate);
       const currentDate = new Date();
-      const ageDiff = new Date(currentDate.setFullYear(currentDate.getFullYear() - MINIMUM_AGE));
+      const minimumAllowedBirthDate = new Date(currentDate.setFullYear(currentDate.getFullYear() - MINIMUM_AGE));
```

Comment:
```suggestion
      const minBirthDate = new Date(currentDate.setFullYear(currentDate.getFullYear() - MINIMUM_AGE));
```
But I suggest to calculate `age` and compare with `MIN_AGE` below.

Reply:
I still don't understand how we can make it much easier than this example